src/registrar/models/contact.py

```diff
@@ -54,6 +54,46 @@ class Contact(TimeStampedModel):
         db_index=True,
     )
 
+    def _get_all_relations(self):
+        return [f.name for f in self._meta.get_fields() if f.is_relation]
```

Comment:
Missing a comment on this function definition - may be handy to have for future devs!

Reply:
fixed, thanks